### ISSUE TITLE
Cant view pinned tabs

### ISSUE THREAD

**Issue by DanaMW** [moneytoo/DontCloseWindowWithLastTab#1]
Your extension would not allow me to view  the pinned tab in the first position. It was a tab of mine and was only affected when pinned.

**Comment by moneytoo** [comment]
This is indeed how it works currently. Initially I was checking the pinned tab location (against ``chrome://newtab/``) but in some rare cases (during startup) I got location of google.com something something instead. It was more than on one occasion so I switched to working with any first pinned tab.

I would definitely like to revisit this issue when I will have more time. I was also thinking about giving users control of what url would be pinned (like ``chrome://bookmarks/``, ``chrome://downloads/`` or anything else) and making it selectable.
